New log file opened at Wed Jan 25 15:45:51 EST 2017 by 6888@akzysk-W370ST
Taking next connection to: 53886
New client attempting to connect on port: 53887-53886
  Address = /192.168.56.1:53886
This client on port 53887 has uuid fcc8f4ac-74c7-48f9-be6e-66d44c3f387e configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-fcc8f4ac-74c7-48f9-be6e-66d44c3f387e
Taking next connection to: 53886
New client attempting to connect on port: 53888-53886
  Address = /192.168.56.1:53886
Handshake exception on socket: 53888
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53886
New client attempting to connect on port: 53889-53886
  Address = /192.168.56.1:53886
This client on port 53889 has uuid fed03592-7f92-4eee-93a1-101852ecfd3a configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-fcc8f4ac-74c7-48f9-be6e-66d44c3f387e, activator-fed03592-7f92-4eee-93a1-101852ecfd3a
Taking next connection to: 53886
New client attempting to connect on port: 53931-53886
  Address = /192.168.56.1:53886
Handshake exception on socket: 53931
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 53886
New client attempting to connect on port: 53932-53886
  Address = /192.168.56.1:53886
This client on port 53932 has uuid 2e0bbf93-3a0c-4d0b-b980-dcdc92a99e5d configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-fed03592-7f92-4eee-93a1-101852ecfd3a, play-fork-run-2e0bbf93-3a0c-4d0b-b980-dcdc92a99e5d
Taking next connection to: 53886
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53886
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53886
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 53886
Checking to see if clients are empty...
No non-daemon clients connected after 3 min. Shutting down.
1 daemon clients are connected.
Server socket thread exiting.
1 clients open, will close them...
All client sockets have been closed.
Closing the 6888@akzysk-W370ST logs at Wed Jan 25 16:00:28 EST 2017, goodbye.
